src/apps/allbridge/shortcuts.ts

```diff
@@ -85,6 +85,40 @@ const hook: ShortcutsHook = {
           return transactions
         },
       }),
+      createShortcut({
+        id: 'withdraw',
+        name: 'Withdraw',
+        description: 'Withdraw your assets',
+        networkIds: [networkId],
+        triggerInputShape: {
+          token: z.object({
+            decimals: z.coerce.number(),
+            amount: z.string(),
+          }),
+          positionAddress: ZodAddressLowerCased,
+        },
+        async onTrigger({ networkId, address, token, positionAddress }) {
+          const walletAddress = address as Address
+          const transactions: Transaction[] = []
+
+          const amountToWithdraw = parseUnits(token.amount, token.decimals)
```

Comment:
Just noting: the logic here will have to match what `withdrawTokenId` we set on the position.

https://github.com/valora-inc/hooks/blob/bc9d1b522ca9ba09d4d94f0af14a8c4619ebc60a/src/types/positions.ts#L68

Users will enter in fiat amount probably?

cc @finnian0826 @satish-ravi

Reply:
@jeanregisser I think the current plan is to not allow withdrawing custom amounts so we would just pass the full token amount, but even if amount is entered in fiat we can easily convert to token amount in the wallet right?